### PR TITLE
fixed macros data migration

### PR DESCRIPTION
**Pull request by alxngyn**
fixes issue #157 

## Changes in this PR.

- [X] added totals to data migration in macro import script

## Testing this PR.

1. Do `docker-compose up`
2. hit `http://localhost:8000/streamwebs/sites/greasy-creek/macros/` 
3. look at time graph

### Expected Output.

```
$ ./runtests # should pass
```
download 2

@osuosl/devs

### CODE REVIEW

**Comment by athai** [comment]
Looks pretty good but the text overlaps when the lines are too close. Can you add a legend?
overlap

File: data_scripts/get_macros.py [hunk 1]
```diff
@@ -64,6 +64,8 @@
             macros.stonefly = row[10]
             macros.water_penny = row[11]
             macros.dobsonfly = row[12]
+            for i in range(7, 12+1):
+                macros.sensitive_total += int(row[i])
```
Comment (athai) [inline data_scripts/get_macros.py]: Are you just adding the values together to get the macro totals? If so, the ``sensitive_total`` and ``somewhat_sensitive_total`` should be weighted.

Comment (alxngyn) [inline data_scripts/get_macros.py]: @athai good catch, fixed!

**Comment by alxngyn** [comment]
legend added 
download 3

**Comment by athai** [comment]
Lgtm. +1

**Comment by leian7** [comment]
+1

**Pull request merged**